components/Highlight/Highlight.js

```diff
@@ -15,4 +16,14 @@ export default function(props){
             </div>
         );
     
-}
\ No newline at end of file
+};
+
+Highlight.propTypes = {
+  
+    image: PropTypes.oneOfType([

```

Comment:
i think its unnecessary to add PropTypes to component that haven't any props yet

Reply:
Highlight has 1 image property that load an image which imported in 'index.js' for Highlight component. I think the <img> tag is minimized on the 'change' tab Cause the code's line jump from line 7 to line 16.

Reply:
okay, good to merge then